source/develop/release-management/features/network/managed_mtu_for_vm_networks.html.md

```diff
@@ -291,6 +308,42 @@ ansible-playbook -i /usr/share/ovirt-engine-metrics/bin/ovirt-engine-hosts-ansib
 5. Use the vNIC profile of this new logical network in a vNIC of the desired VM.
 6. [If requirements are fulfilled, libvirt tries to propagate the MTU into the VM,](#set-the-mtu-by-libvirt). If a subnet is defined for the network, the [MTU is offered via DHCP to the VM](#set-the-mtu-by-dhcp).
 
+#### AutoSync Flow
+
+This is an example flow where the MTU is set by the external provider.
+1. Create a new external logical network with [default MTU](#default-value-for-mtu)
+   on a provider.
+2. If the provider supports the [MTU extenstion][16], it will expose the MTU
+   that is guaranteed to pass through the data path of the segments in the
+   network.
+   During the [automatic synchronization][18] the MTU in the provider's network
+   representation is applied to oVirt Engine's representation.
+3. Use the vNIC profile of this logical network in a vNIC of the desired VM.
+4. [If requirements are fulfilled, libvirt tries to propagate the MTU into the VM,](#set-the-mtu-by-libvirt).
+
+#### AutoDefine Flow
+
+This is an example flow about the MTU is deduced from a host network.
+1. In webadmin, create a new logical network in a data center with a cluster
+   with OVS switch type and a default network provider. You may specify a
+   non-default MTU during creating the network.
+2. A new external network is [created automatically][19] with the original
+   network as its physical network. This external network is created with the
+   same MTU like its physical network. If the physical network was created
+   with default MTU, e.g. 1500, the MTU of the external network will be 1500,
+   too. Please note that this is not the default value for tunneled networks,
+   which is usually 1442 or 1450.
```

Comment:
This could leave the user wondering what he should do in such a case (1500 vs 1442).

Reply:
done